psi4/driver/procedures/proc.py

```diff
@@ -2172,8 +2172,14 @@ def run_ccenergy(name, **kwargs):
     if core.get_global_option('RUN_CCTRANSORT'):
         core.cctransort(ref_wfn)
     else:
-        core.transqt2(ref_wfn)
-        core.ccsort()
+        # try:
+        from psi4.driver.pasture import addins
```

Comment:
I added this try/except block to gracefully deal with some kind of botched install. I commented it out because the raised exception was masking any errors I had from problems linking the ccsort/transqt2.so files to core.so.  I think its okay to remove it all together because if pasture is not installed using the run_cctransort option will throw before we get down here. 

Please try and break this. If you find a case where this line is causing an import error let me know.